versions.tf

```diff
@@ -18,9 +18,16 @@ terraform {
   required_version = ">= 0.13"
   required_providers {
     google = {
-      source = "hashicorp/google"
-      # TODO: Check why this filter was breaking tests.
-      # version = "~> 3.53, < 5.0"
+      source  = "hashicorp/google"
+      version = "~> 4.0"
```

Comment:
```suggestion
      version = ">= 3.53, < 5.0"
```

Reply:
Looks like there was a typo in the commented out version, I suggest this as to not be a breaking change.